### PR TITLE
ci: fixes check tag to ignore semver additions

### PR DESCRIPTION
**Pull request by mathewmeconry**
## Description

Please include a summary of the change and be sure you follow the contributions rules we do provide [here](./CONTRIBUTIONS.md)

Task ID: [OS-?](https://aragonassociation.atlassian.net/browse/OS-?)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.

### CODE REVIEW

File: .github/workflows/reusable-check-tag.yml [hunk 1]
```diff
@@ -32,6 +32,6 @@ jobs:
           GITHUB_REF="${{ inputs.ref }}"
           TAG=${GITHUB_REF##*/}
           VERSION=$(echo "$TAG" | grep -woP "([0-9]+\.[0-9]+\.[0-9]+)-\w+" | sed -E 's/-(sdk|subgraph|contracts|configs)//')
```
Comment (Michael-A-Heuer) [inline .github/workflows/reusable-check-tag.yml:34]: Please write a comment what the commands are doing.

Comment (mathewmeconry) [reply .github/workflows/reusable-check-tag.yml]: was already there but improved with the latest commit

File: .github/workflows/reusable-check-tag.yml [hunk 2]
```diff
@@ -26,12 +26,12 @@ jobs:
       - name: Get version from Github Tag
```
Comment (Michael-A-Heuer) [inline .github/workflows/reusable-check-tag.yml:26]:
```suggestion
      - name: Get version from git Tag
```

**Review by Michael-A-Heuer — APPROVED** [review]
LGTM

Thanks a lot

**Pull request merged**